examples/chip-tool/gen/CHIPClusters.cpp

```diff
@@ -4511,63 +4511,66 @@ CHIP_ERROR OnOffCluster::ReadAttributeClusterRevision(Callback::Cancelable * onS
 }
 
 // OperationalCredentials Cluster Commands
-CHIP_ERROR OperationalCredentialsCluster::GetFabricId(Callback::Cancelable * onSuccessCallback,
-                                                      Callback::Cancelable * onFailureCallback)
+CHIP_ERROR OperationalCredentialsCluster::RemoveFabric(Callback::Cancelable * onSuccessCallback,
+                                                       Callback::Cancelable * onFailureCallback, chip::FabricId fabricId,
+                                                       chip::NodeId nodeId, uint16_t vendorId)
 {
 #if CHIP_ENABLE_INTERACTION_MODEL
     VerifyOrReturnError(mDevice != nullptr, CHIP_ERROR_INCORRECT_STATE);
     (void) onSuccessCallback;
     (void) onFailureCallback;
 
-    app::CommandPathParams cmdParams = { mEndpoint, /* group id */ 0, mClusterId, kGetFabricIdCommandId,
+    app::CommandPathParams cmdParams = { mEndpoint, /* group id */ 0, mClusterId, kRemoveFabricCommandId,
                                          (chip::app::CommandPathFlags::kEndpointIdValid) };
     app::Command * ZCLcommand        = mDevice->GetCommandSender();
 
     ReturnErrorOnFailure(ZCLcommand->PrepareCommand(&cmdParams));
 
-    // Command takes no arguments.
+    TLV::TLVWriter * writer = ZCLcommand->GetCommandDataElementTLVWriter();
+    uint8_t argSeqNumber    = 0;
+    // fabricId: fabricId
```

Comment:
should fabricID type be expanded somehow to specify size? Otherwise this and the nodeid comment below seem not to be too helpful.

a least vendorid states int16u (although unsure if that provides too much help either ... maybe these comments should just go away).

Reply:
This is the autogenerated code. It specifies nodeId and fabricId because those are their own types and elsewhere that is mentioned to be uint64_t. vendorId mentions uint16_t because it not defined as a type anymore. We can have a followup issue filed to fix some of these comments up.